src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -338,7 +338,6 @@ - (void)subscribeWithQueue:(dispatch_queue_t)queue
                                       auto attributePath = std::make_unique<AttributePathParams>();
                                       auto eventPath = std::make_unique<EventPathParams>();
                                       // We want to get event reports at the minInterval, not the maxInterval.
-                                      eventPath->mIsUrgentEvent = true;
```

Comment:
This needs to get set from `params`, no?  Something like:
````
    eventPath->mIsUrgentEvent = params.reportEventsUrgently;
```

otherwise this is completely losing the urgent-event bits in this function...

Reply:
I'm just going to fix this up, in the interests of getting this merged.